js/ui/date_box/ui.date_box.base.js

```diff
@@ -62,6 +62,10 @@ const DateBox = DropDownEditor.inherit({
         return extend(this.callBase(), this._strategy.supportedKeys());
     },
 
+    _shouldBeClosed() {
```

Comment:
sounds too common as for me, let's name it smth like `_shouldCloseOnTab` for better understanding

Reply:
Nice point) Fixed in [99a178d2a43ec5be8ad8dd319a440caaa9118077].